### PR TITLE
Ask user for his name on startup

### PR DESCRIPTION
**Pull request by ketanbhatt**
This serves two purposes:
1. code review
2. You get to see/know what I did

Focus is not on the UI right now. But if you see something done inefficiently, point it out.

https://github.com/ketanbhatt/habits/issues/3

PS: You can start by cloning the repository and doing `npm install` and `npm start`. Assuming you have `node` pre installed

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+const electron = require('electron');
+const ipc = electron.ipcMain;
+const app = electron.app;  // Module to control application life.
```
Comment (WhiteHatArpit) [inline main.js:5]: Use `const app = require('app');` instead.
Same for all other `electron.` statements.

**Comment by WhiteHatArpit** [comment]
Code style needs to be fixed.

File: main.js (hunk 1 above)
Comment (ketanbhatt) [inline main.js:5]: I am not sure, this I took from electron's official Quick Guide. Can you double check which convention is to be used?

**Comment by ketanbhatt** [comment]
Will pull develop after we merge https://github.com/ketanbhatt/habits/pull/12

File: main.js (hunk 1 above)
Comment (WhiteHatArpit) [inline main.js:5]: https://github.com/atom/electron/issues/3708
Supposedly, a global installation of electron-prebuilt causes trouble when using `electron.app` convention. Removing it, things got fine. `require('app')` worked either way.
Lets go with the Quick Guide convention itself, the one you have used.

**Comment by WhiteHatArpit** [comment]
After pressing the 'Enter Name' button, the settings window gets closed, the home window gets drawn but it gets closed itself in the split second.

**Comment by ketanbhatt** [comment]
That didn't happen when I did it. Wait till i pull master and then test.

**Pull request merged**